src/network/network_gui.cpp

```diff
@@ -283,18 +283,20 @@ class NetworkGameWindow : public Window {
 		return (r != 0) ? r < 0 : NGameClientSorter(a, b);
 	}
 
-	/** Sort servers by current date */
-	static bool NGameDateSorter(NetworkGameList * const &a, NetworkGameList * const &b)
+	/** Sort servers by calendar date. */
+	static bool NGameCalendarDateSorter(NetworkGameList * const &a, NetworkGameList * const &b)
 	{
-		auto r = a->info.game_date - b->info.game_date;
+		auto r = a->info.calendar_date - b->info.calendar_date;
 		return (r != 0) ? r < 0 : NGameClientSorter(a, b);
 	}
 
-	/** Sort servers by the number of days the game is running */
-	static bool NGameYearsSorter(NetworkGameList * const &a, NetworkGameList * const &b)
+	/** Sort servers by the number of ticks the game is running. */
+	static bool NGameRunningDateSorter(NetworkGameList * const &a, NetworkGameList * const &b)
```

Comment:
Maybe duration is a better name than date? Maybe even `NGameDurationSorter`?
```suggestion
	static bool NGameRunningDurationSorter(NetworkGameList * const &a, NetworkGameList * const &b)
```